rules/eslint/suggestions.js

```diff
@@ -0,0 +1,213 @@
+module.exports = {
+  rules: {
+    'accessor-pairs': 'error',
+    'arrow-body-style': ['error', 'as-needed'],
```

Comment:
I tend to disagree with required braces because it eliminates the whole point of arrow functions that they do not require `return` statement to return values. It also would cause a lot of changes in Hilla codebase. So I would suggest to just override this rule for your configuration separately.

Actually, looking at your example I still in favor of using arrow functions here, because I don't see any specific difference between two options you provided. The second option looks better for me, I would write it this way. Though, the pain point with the big changelog remains.